src/libcore/iter.rs

```diff
@@ -2246,8 +2246,8 @@ impl<I> RandomAccessIterator for Fuse<I> where I: RandomAccessIterator {
 impl<I> ExactSizeIterator for Fuse<I> where I: ExactSizeIterator {}
 
 impl<I> Fuse<I> {
-    /// Resets the fuse such that the next call to .next() or .next_back() will
-    /// call the underlying iterator again even if it previously returned None.
+    /// Resets the `Fuse` such that the next call to `.next()` or `.next_back()` will call the
+    /// underlying iterator again even if it previously returned `None`.
```

Comment:
Could you keep this comment wrapped at 80 characters as well? (the surrounding style)